torchci/lib/bot/autoLabelBot.ts

```diff
@@ -175,7 +176,28 @@ function getRepoSpecificLabels(
   return repoSpecificAutoLabels[repoKey];
 }
 
+function TDRolloutIssueParser(rawSubsText: string): object {
+  const subsText = rawSubsText.replace("\r", "");
+  const subsRows = subsText.match(/^\*.+/gm);
+  const authors: any = new Set();
+  if (subsRows == null) {
+    return authors;
+  }
+  subsRows.forEach((row: string) => {
+    const users = row.match(/@[a-zA-Z0-9-/]+/g);
```

Comment:
[This](https://github.com/pytorch/pytorch/issues/123120) is the issue it parses, right?  I'm not seeing "@" before a person's name. 

Does the issue need to be fixed?

Reply:
fixed